.clang-format

```diff
@@ -0,0 +1,22 @@
+BasedOnStyle: LLVM
+ColumnLimit: 100
+UseTab: Never
+TabWidth: 2
+IndentWidth: 2
+IncludeBlocks: Preserve
+NamespaceIndentation: None
+PointerAlignment: Right
+AlignAfterOpenBracket: AlwaysBreak
+AlignEscapedNewlines: DontAlign
+AllowAllParametersOfDeclarationOnNextLine: false
+PenaltyReturnTypeOnItsOwnLine: 300
+BinPackArguments: false
+BinPackParameters: false
+IndentCaseLabels: true
+BreakBeforeTernaryOperators: true
+BreakBeforeBinaryOperators: All
```

Comment:
since there was opinions about the brace on a new line in the `sdk_for_c` can we add another option here to break before opening brace?

`BreakBeforeBraces: Allman`

to always break

Reply:
I would prefer to keep it as is for now to avoid another deviation from the Azure SDK Guidelines.

---
In reply to: [338255804](https://github.com/Azure/azure-ulib-c/pull/40#discussion_r338255804) [](ancestors = 338255804)